components/bl-PhotoSwipeGallery/bundle.js

```diff
@@ -0,0 +1,144 @@
+define(() => {
+    
+  function initLibrary(options) {
+    if (!options.separator) {
+      options.separator = 'of';
+    }
+    
+    const styleTag = !document.getElementById("photoswipeStyle");
```

Comment:
`const styleTagExists = !!document.getElementById("photoswipeStyle")`? 
This is a boolean variable, but `styleTag` looks like there is some element inside it.
And then it looks strange - "if scriptTag or styleTag - create tags..."